develop-guide/writetrigger/writetrigger.rst

```diff
@@ -661,17 +657,17 @@ somewhere in the output, irrelevant of the structure.
 
         # Can have as many items under include or exclude that you want
         include:
-            - key: "[version][version]"
-              value: 16.9.1
-            - key: "[version][main_mem]"
+            # Using Dq (the new dictionary querying tool), you can verify whether 

```

Comment:
provide link to dq doc

https://pubhub.devnetcloud.com/media/genie-docs/docs/userguide/utils/index.html#dq

Reply:
I did add the link, but this is in the code block and it looks weird to me. I actually below mentioned Dq and gave link to it.